main.go

```diff
@@ -12,13 +12,21 @@ import (
 
 func makeHandler(name string, fn func(http.ResponseWriter, *http.Request, string, string)) http.HandlerFunc {
```

Comment:
Great, thanks for confirming that. I was only 80% confident that was the case. :)